app/lib/server/routers/docs.router.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const routesAbstractionController = require('../controllers/RoutesAbstraction.controller.js').getInstance();
+
+module.exports = {
+    routesAbstractionController,
```

Comment:
I Think `abstraction` is not the right word for this. 
I would rather use `reflection`, or more simply `apiDocumentation` (this applies to controller, routes, description and so on)